pixiu/dubbo/go-server/pkg/user.go

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
May be all files need this license, consistent with the project configuration.

Reply:
This file is migrated from apache/dubbo-go-pixiu, so you need to add a license